docs/db2graph/db2graph.rst

```diff
@@ -0,0 +1,290 @@
+Db2Graph: Database to Graph conversion tool
+============================================
+
+Introduction
+""""""""""""""""""""
+
+**Db2Graph** converts **relational databases** into **graphs as sets of triples** which can be used as **input datasets for Marius**, allowing streamlined preprocessing from database to Marius. Db2Graph comes with Marius but can be used as a standalone tool. Conversion with Db2Graph is achieved in the following steps: 
+
+#. Users import/create the database locally
+
+#. Users define the configuration file and entity/edge SQL SELECT queries
+
+#. Db2Graph executes the SQL SELECT queries
+
+#. Db2Graph transforms the result set of queries into sets of triples
+
+Below we lay out the requirements, definitions, and steps for using Db2Graph, and a real example use case:
+
+Requirements
+""""""""""""""""""""
+
+Db2Graph currently supports graph conversion from three relational database management systems: **MySQL**, **MariaDB**, and **PostgreSQL**. Db2Graph requires no installation and all the required python packages are part of Marius installation. Following are the packages required:
+
+* python >= 3.6  (included in Marius installation)
```

Comment:
Can remove this list and just say it requires Marius to be installed with pip and link to the corresponding doc page

Reply:
Updated